src/main/java/us/kilroyrobotics/Robot.java

```diff
@@ -38,13 +56,16 @@ public void robotPeriodic() {
          * of how to use vision should be tuned per-robot and to the team's
          * specification.
          */
-        if (kUseLimelight) {
+        if (VisionConstants.kUseLimelight) {
             var driveState = m_robotContainer.drivetrain.getState();
             double headingDeg = driveState.Pose.getRotation().getDegrees();
             double omegaRps = Units.radiansToRotations(driveState.Speeds.omegaRadiansPerSecond);
 
-            LimelightHelpers.SetRobotOrientation("limelight", headingDeg, 0, 0, 0, 0, 0);
-            var llMeasurement = LimelightHelpers.getBotPoseEstimate_wpiBlue_MegaTag2("limelight");
+            LimelightHelpers.SetRobotOrientation("limelight-right", headingDeg, 0, 0, 0, 0, 0);
+            LimelightHelpers.SetRobotOrientation("limelight-left", headingDeg, 0, 0, 0, 0, 0);
+            var llMeasurement = LimelightHelpers.getBotPoseEstimate_wpiBlue("limelight-right");
+            if (llMeasurement == null)
```

Comment:
Explain what this is doing. I know what it is doing, but add a comment explaining